.circleci/config.yml

```diff
@@ -174,6 +174,8 @@ commands:
           command: |
             ./test/runner << parameters.test_targets >>
             $EMSDK_PYTHON ./tools/check_clean.py
+      - store_test_results:
+          path: test-results.xml
```

Comment:
How about `out/test-results.xml`..  I've been thinking about designating and using `out/` as place to put test artifacts.